src/form/AsyncSelect/AsyncSelect.stories.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { ComponentStory, ComponentMeta } from '@storybook/react';
+import { AsyncSelect } from './AsyncSelect';
+
+type TypeaheadProps = {
+	_display: string;
```

Comment:
What's the meaning of the prefixed underscore?

Reply:
I will remove this and replace it withj a __stubs__ , the data.amsterdam just returns data that way :(